mailpoet/mailpoet
# Allow multiple values in purchased in this category segment - MAILPOET-3953

[MAILPOET-3953](https://mailpoet.atlassian.net/browse/MAILPOET-3953)

Testing instructions:

1. Go to WP Admin > MailPoet > Lists
2. Click on Segments tab
3. Click to `+ New Segment` button at the top
4. From there, test the segment creation and modification according to [MAILPOET-3953](https://mailpoet.atlassian.net/browse/MAILPOET-3953) specs in Jira ticket. This change is related to `purchased in category` type of a segment.
5. Make sure you follow the specs and that it behaves according to the specification.
